src/modules/75-cf/RouteDestinations.tsx

```diff
@@ -49,24 +48,107 @@ import CreateSecretFromYamlPage from '@secrets/pages/createSecretFromYaml/Create
 import RbacFactory from '@rbac/factories/RbacFactory'
 import { ResourceCategory, ResourceType } from '@rbac/interfaces/ResourceType'
 import { PermissionIdentifier } from '@rbac/interfaces/PermissionIdentifier'
+import featureFactory from 'framework/featureStore/FeaturesFactory'
+import { FeatureIdentifier } from 'framework/featureStore/FeatureIdentifier'
+import { BannerType } from '@common/layouts/Constants'
 import { String } from 'framework/strings'
 import { useFeatureFlags } from '@common/hooks/useFeatureFlag'
 import GitSyncPage from '@gitsync/pages/GitSyncPage'
 import GitSyncRepoTab from '@gitsync/pages/repos/GitSyncRepoTab'
 import GitSyncEntityTab from '@gitsync/pages/entities/GitSyncEntityTab'
 import GitSyncErrors from '@gitsync/pages/errors/GitSyncErrors'
 import GitSyncConfigTab from '@gitsync/pages/config/GitSyncConfigTab'
+import { formatToCompactNumber } from '@cf/utils/CFUtils'
 import { TargetsPage } from './pages/target-management/targets/TargetsPage'
 import { TargetDetailPage } from './pages/target-details/TargetDetailPage'
 import { SegmentsPage } from './pages/target-management/segments/SegmentsPage'
 import { SegmentDetailPage } from './pages/segment-details/SegmentDetailPage'
 import TargetGroupDetailPage from './pages/target-group-detail/TargetGroupDetailPage'
 import { OnboardingPage } from './pages/onboarding/OnboardingPage'
+import UsageLimitBannerText from './components/UsageLimitBanner/UsageLimitBannerText'
 
 import { OnboardingDetailPage } from './pages/onboarding/OnboardingDetailPage'
 import CFTrialHomePage from './pages/home/CFTrialHomePage'
 import FeatureFlagsLandingPage from './pages/feature-flags/FeatureFlagsLandingPage'
 
+featureFactory.registerFeaturesByModule('cf', {
+  features: [FeatureIdentifier.MAUS],
+  renderMessage: (props, getString, additionalLicenseProps = {}) => {
+    const { isEnterpriseEdition, isFreeEdition, isTeamEdition } = additionalLicenseProps
+    const featuresMap = props.features
+    const monthlyActiveUsers = featuresMap.get(FeatureIdentifier.MAUS)
```

Comment:
can you merge line `78/79`?

```suggestion
    const monthlyActiveUsers = props.features.get(FeatureIdentifier.MAUS)
```

Also this number could be `undefined` (for whatever reason) so might want to check for that. I think we cater for 0 below so thats good

Reply:
done